cmd/fyne/internal/templates/data/Makefile

```
@@ -6,6 +6,11 @@ LOCAL ?= $(shell test -d $(DESTDIR)/usr/local && echo "/local" || echo "")
 PREFIX ?= /usr$(LOCAL)
 
 default:
+	# Install for simple user
```

This seems wage and also like it could almost sound a bit rude. What is a "simple user"?
How about something like this instead (feel free to suggest something else if you like):
```suggestion
	# User install
```